openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -338,6 +338,24 @@ export default function init(){
             $('.wmd-preview').before('<h3 id="prevHead" style="margin:15px 0 10px;padding:0;">Preview</h3>');
         }
     });
+
+    // For Book Preview Button
+    $(document).ready(function(){
```

Comment:
No need for using .ready here.

Can this change wait for
https://github.com/internetarchive/openlibrary/pull/2305/commits/cf6527e30ec3b6afe9a923ca93e01218dce0bdca which is blocked on code review on https://github.com/internetarchive/openlibrary/pull/2344 

If those patches can be prioritised this will save me a lot of work in future and we can restrict this code to only the pages that need it.

Reply:
done